config/_default/menus/menus.en.yaml

```diff
@@ -2605,12 +2605,84 @@ main:
     parent: ci
     identifier: ci_troubleshooting
     weight: 10
+  - name: CD Visibility
+    url: continuous_delivery/
+    pre: ci
+    identifier: cd
+    parent: ci_visibility_heading
+    weight: 160000
+  - name: Pipeline Visibility
+    url: continuous_delivery/pipelines
+    identifier: cd_pipelines
+    parent: cd
+    weight: 1
+  - name: ArgoCD
+    url: continuous_delivery/pipelines/argocd
+    identifier: cd_pipelines_argocd
+    parent: cd_pipelines
+    weight: 101
+  - name: Search and Manage
+    url: continuous_delivery/search
+    identifier: cd_search
+    parent: cd
+    weight: 2
+  - name: Explorer
+    url: continuous_delivery/explorer
+    identifier: cd_explorer
+    parent: cd
+    weight: 3
+  - name: Search Syntax
+    url: continuous_delivery/explorer/search_syntax
+    identifier: cd_explorer_search_syntax
+    parent: cd_explorer
+    weight: 301
+  - name: Facets
+    url: continuous_delivery/explorer/facets
+    identifier: cd_explorer_facets
+    parent: cd_explorer
+    weight: 302
+  - name: CD Visibility
+    url: continuous_delivery/
+    pre: ci
+    identifier: cd
+    parent: ci_visibility_heading
+    weight: 160000
+  - name: Pipeline Visibility
+    url: continuous_delivery/pipelines
+    identifier: cd_pipelines
+    parent: cd
+    weight: 1
+  - name: ArgoCD
+    url: continuous_delivery/pipelines/argocd
+    identifier: cd_pipelines_argocd
+    parent: cd_pipelines
+    weight: 101
+  - name: Search and Manage
+    url: continuous_delivery/search
+    identifier: cd_search
+    parent: cd
+    weight: 2
+  - name: Explorer
+    url: continuous_delivery/explorer
+    identifier: cd_explorer
+    parent: cd
+    weight: 3
+  - name: Search Syntax
+    url: continuous_delivery/explorer/search_syntax
+    identifier: cd_explorer_search_syntax
+    parent: cd_explorer
+    weight: 301
+  - name: Facets
+    url: continuous_delivery/explorer/facets
+    identifier: cd_explorer_facets
+    parent: cd_explorer
+    weight: 302
```

Comment:
@alai97 Was it intentional to have duplicated elements here? For example the "Facets" configuration defined in lines 2675-2679 is the same of the one defined in lines 2639-2643